### DISCUSSION TITLE
request driver support for Google Pixel 6, CPU Tensor, GPU Mali

### DISCUSSION THREAD

**Discussion by hdk2d** [skyline-emu/skyline#2182] [Q&A] [answered]
will driver support for Mali be available in the future?

edit:
GPU

**Comment by JERD001** [comment, marked as answer]
Not really, pretty unlikely to see it in the near future. Only hope are reverse engineering projects but they don't have Vulkan to the level necessary for Skyline.

**Comment by Crytonics** [comment]
Rn there is no custom drivers for mali gpu, only for Adreno gpu's.

**Reply by hdk2d** [reply]
I don't know if it's for marketing or what but it's already very limiting for me to change device it takes me a year and this device I got recently if CPU was changed like in PC it would solve the problem
